scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -357,6 +495,20 @@ private static CommandLine getCommandLine(String[] args, String loginFilePath, S
         opt.setRequired(false);
         options.addOption(opt);
 
+        opt = new Option(SOURCE_LOGINFILE_OPTION, SOURCE_LOGINFILE_OPTION_NAME, true,
+                "Path to a source login file, used for bulk creation or bulk update. The source login file must contain clear text passwords in the format username:password");
+        opt.setArgName(SOURCE_LOGINFILE_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option(SOURCE_GROUPFILE_OPTION, SOURCE_GROUPFILE_OPTION_NAME, true,
+                "Path to a source group file, used for bulk creation or bulk update. The source group file must contain group assignements in the format username:group");
+        opt.setArgName(SOURCE_GROUPFILE_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        opt.setRequired(false);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)